[#147] rename tagged to tags

Eclipse-Dominator opened the pull request:
Fixes #147 

Change the variable tagged to tags in the affected class [took over from #153]
Change the json property tagged to tags in all affected save files
(src/main/java/seedu/address/storage/JsonAdaptedPerson.java).

![image](https://github.com/se-edu/addressbook-level3/assets/4567895/20855574-cc1a-414e-8a71-fc7b8292aa9e)

damithc commented on gradlew:
```diff
@@ -28,7 +28,7 @@ APP_NAME="Gradle"
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS=""
+DEFAULT_JVM_OPTS='"-Xmx64m"'
```
Do we need to explain this change in one of the commit messages?

pyokagan replied:
It's explained:

```
The Gradle wrapper upgrade is performed by running the command:

    ./gradlew wrapper --gradle-version 5.2.1
```

Py0000 committed:
JsonAdaptedPerson: Change variable name tagged -> tags

The JsonAdaptedPerson class contains a list object called 'tagged'.

According to our coding standard, a variable holding a collection
of items should have a plural form name.

Let's change the variable name to 'tags' instead.

damithc commented on src/main/java/seedu/address/storage/JsonAdaptedPerson.java:
```diff
@@ -54,19 +54,21 @@ public JsonAdaptedPerson(Person source) {
     /**
-     * Converts this Jackson-friendly adapted person object into the model's {@code Person} object.
+     * Converts this Jackson-friendly adapted person object into the model's
+     * {@code Person} object.
```
Unrelated change?

Eclipse-Dominator replied:
I have updated my formatter to only format source changes.

damithc commented on src/main/java/seedu/address/storage/JsonAdaptedPerson.java:
```diff
@@ -54,19 +54,21 @@ public JsonAdaptedPerson(Person source) {
      *
-     * @throws IllegalValueException if there were any data constraints violated in the adapted person.
+     * @throws IllegalValueException if there were any data constraints violated in
+     *                               the adapted person.
```
Unrelated changed?

damithc reviewed (changes requested):
1. Some minor nits suggested.
1. The new commit is missing a full commit message, compliant with https://se-education.org/guides/conventions/git.html#commit-message-body
1. Did you do some manual tests to verify the tagging still works?

Eclipse-Dominator commented:
I have updated the commit to include a commit body. I did do some testing with changes and no issues were spotted.

damithc reviewed (changes requested):
Code looks fine now. The commit messages can be improved further.

```
JsonAdaptedPerson class: Change tagged to tags
........................
JsonAdaptedPerson class has a list object that is named tagged.

A list object represents a collection of objects hence should be in
plural form based on the coding standard.

Let's change the name of the list object from tagged to tags.
```

```
JsonAdaptedPerson: Use tags for JSON property (#169)
........................
Currently, "tagged" is used as the default JSON attribute when saving.

It is inconsistent with the new naming scheme.

Let's change the JSON attribute to be consistent with
the new naming scheme.
```

* The two commit subjects should be phrased similarly, as they are doing similar things.
* `Currently` can be omitted, as it is implied.
* These phrases raise questions `default JSON attribute` (_default_?), `new naming scheme` (what _new_ naming scheme?)

How  about the following?
```
JsonAdaptedPerson: Change variable name tagged -> tags
...............
The JsonAdaptedPerson class contains a list object called 'tagged'.

According to our coding standard, a variable holding a collection
of items should have a plural form name. 

Let's change the variable name to 'tags' instead.
```

```
JsonAdaptedPerson: Change JSON property name tagged -> tags
......................
Similar to the previous commit, let's change the JSON property name 
'tagged' in the JsonAdaptedPerson class to 'tags'.
```

Eclipse-Dominator committed:
JsonAdaptedPerson: Change JSON property name tagged -> tags

Similar to the previous commit, let's change the JSON property name
'tagged' in the JsonAdaptedPerson class to 'tags'.

damithc reviewed (changes requested):
Almost there. You missed updating the subject of the commit 1 subject. :-p

Eclipse-Dominator commented:
Yup I just realized it as well.

damithc reviewed (approved):
LGTM 👍

damithc commented:
Thanks to @Py0000 for your contribution to this PR too.
